internal/cmd/auth.go

```diff
@@ -81,12 +81,21 @@ var tokenCmd = &cobra.Command{
 	},
 }
 
+var apiTokensCmd = &cobra.Command{
+	Use:   "api_tokens",
```

Comment:
I don't think `_` in a command name is a good idea. `-` is better IMO

Reply:
Done!